connect-app/app/screens/Home.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import { useNavigation } from "@react-navigation/core";
+import { auth } from "../../FirebaseConfig";
+
+const Home = () => {
+  const navigation = useNavigation();
+
+  const handleSignOut = () => {
+    auth
+      .signOut()
+      .then(() => {
+        navigation.replace("Login");
```

Comment:
Can this be Home? instead of Login. I beleive SignOut leads to Home and we can set login as button in home page. In future we may set a home page and display some content about the app, services, contact etc...

Reply:
Never mind about this comment. I suggested this from a web perspective. So I think this works for mobile.

Reply:
Right now, the flow is Login -> Home 
Logout -> Login page

Reply:
ok